installer/bundle/bundle_skel.sh

```diff
@@ -941,13 +941,31 @@ if [ "$installMode" = "R" -o "$installMode" = "P" ]; then
         fi
 
         pkg_rm omsconfig
-        pkg_rm omsagent
 
-        # If MDSD is installed and we're just removing (not purging), leave SCX
-        MDSD_INSTALLED=1
-        check_if_program_exists_on_system azsec-mdsd
-        if [ $? -eq 0 -o -d /var/lib/waagent/Microsoft.OSTCExtensions.LinuxDiagnostic-*/mdsd ]; then
-            MDSD_INSTALLED=0
+        # If MDSD/LAD is installed and we're just removing (not purging), leave OMS, SCX and OMI
```

Comment:
Pls fix the comment wrt purging. It should not be uninstalled even when purging.

Reply:
We will handle purging on a different PR, let focus here on the remove.

Reply:
ok